wordpress-mobile/WordPress-iOS
# Quick Start: Checklist improvements

Refs #9447

This adds several refinements to the Quick Start checklist that make it feature-complete.

![checklist improvements](https://user-images.githubusercontent.com/517257/47202254-869b8f00-d331-11e8-9362-cba2fc2eb180.jpg)

- Progress count added on blog details screen
- checkmarks added for completed items
- skip all button added (with alert to verify first)
- congratulations header added when entire list has been completed
- also make the suggestion timeout 10s because @SylvesterWilmott told me to

To test:
- Make a new site and turn one Quick Start, or ask me for my snippet to turn it on
- checkout each of the above items
- kinda make sure nothing else in quick start was broken

@ScoutHarris @aerych can one of you check this out tomorrow so it makes the code freeze?
